netsim/ansible/tasks/deploy-config/eos.yml

```diff
@@ -1,4 +1,24 @@
+- set_fact: deployed_config={{ lookup('template',config_template) }}
+- set_fact: bash_script={{ "#!/bin/bash" in deployed_config }}
+
+- template:
+    src: "{{ config_template }}"
+    dest: /tmp/config.sh
+  when: bash_script
+  vars:
+    ansible_user: root
+    ansible_connection: docker
+
+- name: "run /tmp/config.sh to deploy {{netsim_action}} config from {{ config_template }}"
+  command: bash /tmp/config.sh
+  when: not ansible_check_mode and bash_script
+  become: true
+  vars:
+    ansible_user: root
+    ansible_connection: docker
```

Comment:
Because you're relying on Docker connection, it would be best to have this task list as "eos-clab.yml", so it's not used for a vEOS VM.

Reply:
... also, you might need tags on this task to display it within **netlab initial -q** (or whatever the "don't display all the clutter" flag is)